deploy.sh

```diff
@@ -283,10 +283,10 @@ function checkDate() {
     if [ -f $tempFile ]; then
         value=$(< $tempFile) # Time of last beat campaign
         now=$(date +"%s") # Current time
-        let "difference = $now - $value" # Time since last beat campaign
+	now=$((now - value)) # Time since last beat campaign
 
         # If been longer than 3 days, set forceFightCampaign=true
-        if [ $difference -gt 255600 ]; then
+        if [ "$difference" -gt 255600 ]; then
             forceFightCampaign=true
         fi
```

Comment:
Removing the `difference` variable and then attempting to use it on line `289` won't work. You'd have to change `289` to `"$now"`.